### DISCUSSION TITLE
プロジェクトチームによっては、自身の意思を反映させたプロジェクトゴールにならない場合も多いのでは？

### DISCUSSION THREAD

**Discussion by CP-koshikawa** [copilot-jp/project-sprint#382] [Q&A]
URL：https://www.projectsprint.org/ja/v3.3/theories/101
「[プロジェクトチームは、自身の意思を反映させたプロジェクトゴールを設定し共有する。]
クライアントワークの開発ベンダーの場合、「自身の意思」は関係ないこともあると思うので違和感を感じました。
プロジェクトフェーズによっては当てはまるとは思うので、PJSの前提となっているプロジェクトのフェーズはどこかで明示しておいたほうがよいと思います

【実践チーム】文書内容への違和感：Project Sprint 101>コアアクション

**Comment by MasaruSaito** [comment]
自身の意思というのは、プロジェクトチームの意思と理解したので、私は違和感ありませんでした

**Comment by CP-koshikawa** [comment]
プロジェクトゴール→プロジェクトへの期待値
であれば、ベンダー側の意思も知りたいので、違和感がない。

**Comment by motoi** [comment]
プロジェクトゴールが「ユーザーのハピネスを追求する」などシンプルな1つのものになったときは自分たちの意思が入っていないものになるように感じるという話かと。プロジェクトゴールという言葉のとらえ方がちがうのかもしれない。事例があるとわかりやすいかも。

**Comment by kameokamayu** [comment]
「ゴール」が「成果」だと違和感を感じない。

**Comment by CP-koshikawa** [comment]
＜結論＞
こちらのDisscussionのIssuesを立てる【こころさん】
